packages/cf-wallet.js/test/fixture.ts

```diff
@@ -22,19 +24,22 @@ export class TestNodeProvider implements INodeProvider {
     methodName: Node.MethodName,
     callback: (message: Node.MethodRequest) => void
   ) {
-    this.messageEmitter.on(methodName, callback);
+    this.messageEmitter.on(
+      jsonRpcMethodNames[methodName] || methodName,
+      callback
+    );
   }
 
-  public simulateMessageFromNode(message: Node.Message) {
+  public simulateMessageFromNode(message: any) {
```

Comment:
There seem to be many uses of `any` like this; are there no type enforcements to be made?

Reply:
Yeap, working on them.